app/serializers/current_user_serializer.rb

```diff
@@ -312,6 +314,22 @@ def include_experimental_sidebar_enabled?
     SiteSetting.enable_experimental_sidebar
   end
 
+  def sidebar_category_ids
+    object.sidebar_section_links.where(linkable_type: 'Category').pluck(:linkable_id)
```

Comment:
If we decide for previous change, then let's use `object.sidebar_categories.pluck(:linkable_id)`